esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2999,12 +2999,11 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
             if (server.isEmpty())
                 throw MakeStringException(ECLWATCH_INVALID_INPUT, "Invalid server for dropzone %s.", dropZoneName);
 
-            IpAddress ipAddr;
-            ipAddr.ipset(server.str());
-            if (ipAddr.ipequals(ipToMatch))
+            //Do string compare here because the server could be a pseudo-host name.
+            if (strieq(dropZoneServerReq, server))
             {
                 serverFound = true;
-                searchDropZoneFiles(context, ipAddr, directory.str(), nameFilter, files, filesFound);
+                searchDropZoneFiles(context, ipAddress, directory.str(), nameFilter, files, filesFound);
```

Comment:
Feels like we should be passing dropZoneServerReq here - but perhaps that's for the phase 2 ?

Reply:
yes I think it should, but that can be in a separate PR - this is a function for listing files onj a drop zone, which isn't implemented for azure blobs yet